handlers/update_patient.go

```diff
@@ -0,0 +1,37 @@
+package handlers
+
+import (
+	models "github.com/abihaa/hospital-management-system/models"
+	"github.com/go-openapi/runtime/middleware"
+
+	runtime "github.com/abihaa/hospital-management-system"
+	"github.com/abihaa/hospital-management-system/gen/restapi/operations"
+)
+
+// NewUpdatePatient handles update request.
+func NewUpdatePatient(rt *runtime.Runtime) operations.UpdatePatientHandler {
+	return &updatePatient{rt: rt}
+}
+
+type updatePatient struct {
+	rt *runtime.Runtime
+}
+
+func (u updatePatient) Handle(params operations.UpdatePatientParams) middleware.Responder {
+	_, err := u.rt.Service().GetPatientByID(params.ID)
+	if err != nil {
+		return operations.NewGetPatientNotFound()
+	}
+	if err := u.rt.Service().UpdatePatient(&models.Patient{
```

Comment:
add new line for to improve readability. Divide code in logical blocks divided by newline